CHANGELOG.md

```diff
@@ -1,19 +1,25 @@
 # Changelog
 
+## [0.10.1]
```

Comment:
Are we going to make releases manual?

Reply:
This will become a release automatically - see label